### ISSUE TITLE
0.30.0 - `send_command` service call  fails with `Other error`

### ISSUE THREAD

**Issue by zxdavb** [zxdavb/ramses_cc#75]
On **0.30.0**, for:
```yaml
service: remote.send_command  # and now also: ramses_rf.send_command
data:
  command: normal
target:
  entity_id: remote.32_205847
```
I am able to reproduce the following error.

```
2023-11-11 21:33:21.496 ERROR (MainThread) [ramses_rf.gateway] 
Failed to send 22F1| I|32:205847: 7FFF| I|18:000730: Other error
```
Fix coming.

**Comment by zxdavb** [comment]
I think this is fixed, for now, because QoS is currently disabled.

**Comment by zxdavb** [comment]
I believe this is fixed from **0.31.7** (or before that)
